uwb/example/lib/game_widgets/score.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+/// The [Score] class that generates the score.
+class Score extends StatelessWidget {
+  /// The [Score] constructor.
+  const Score({Key? key, required this.score, required this.gameHasStarted})
+      : super(key: key);
+
+  /// Int variable to keep track of the current score.
+  final int score;
+
+  /// Boolean to know if the game is started so the score can be showed.
```

Comment:
Same as with the documentation on the [brickFieldList] field of the [BrickField] class:
```suggestion
  /// Indicates if the game has been started.
```